jdaviz/core/tests/test_tools.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+from glue.core import Data
+
+
+def test_boxzoom(cubeviz_helper, spectral_cube_wcs):
+    data = Data(flux=np.ones((128, 128, 256)), label='Test Flux', coords=spectral_cube_wcs)
+    cubeviz_helper.app.data_collection.append(data)
+
+    cubeviz_helper.app.add_data_to_viewer('spectrum-viewer', 'Test Flux')
+    cubeviz_helper.app.add_data_to_viewer('flux-viewer', 'Test Flux')
+
+    flux_viewer = cubeviz_helper.app.get_viewer('flux-viewer')
+
+    assert flux_viewer.state.y_min == -0.5
+    assert flux_viewer.state.y_max == 127.5
+    assert flux_viewer.state.x_min == -0.5
+    assert flux_viewer.state.x_max == 127.5
+
+    # Test does not work on python 3.8?
```

Comment:
So it was not just my local installation (which is 3.10 on macOS)...

Reply:
It worked on your installation or no?

Reply:
It looks OK in the notebook, but the test fails with
```python
>       t = flux_viewer.toolbar_nested.tools['jdaviz:boxzoom']
E       AttributeError: 'CubevizImageView' object has no attribute 'toolbar_nested'
```

Reply:
Good call! Waiting to see if tests pass now.

Reply:
Works for me!